dev-haskell/strptime/strptime-1.0.10-r1.ebuild

```diff
@@ -1,25 +1,24 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-# ebuild generated by hackport 0.3.4.9999
+# ebuild generated by hackport 0.6.7.9999
 
 CABAL_FEATURES="lib profile haddock hoogle hscolour"
 inherit haskell-cabal
 
-DESCRIPTION="Efficient parsing of LocalTime using a binding to C's strptime"
+DESCRIPTION="Efficient parsing of LocalTime using a binding to C's strptime, with some extra features (i.e. fractional seconds)"
```

Comment:
I think `DESCRIPTION` has a 80 char limit. Does it not introduce new warnings?

Reply:
Yes, this does introduce a new warning.

Per my monologue up top, this update came straight from hackport.

I can think of two approaches to deal with this:

1. Leave it as-is and tackle all the "TooLong" description warnings at once (I was already planning on this)
2. rerun my script and add a check for description length, though honestly I'd expect this check to be done by hackport

Let me know if you have a preference.

Reply:
I'd prefer not to introduce known regressions to `::haskell`. `hackport` always needs a bit of human oversight.

Reply:
Ok, I fixed this and one other `DESCRIPTION` that was too long.

Regarding hackport, I understand and agree that it requires some human oversight, however for something as rote as "DESCRIPTION should be 80 chars or less" I would expect hackport to handle it.

Perhaps I should have incorporated a check in my script for pre-and-post repoman warnings, to see if there are any new ones.